Bio/Pathway/__init__.py

```diff
@@ -35,7 +35,7 @@
 
 from functools import reduce
 
-from Bio.Pathway.Rep.MultiGraph import MultiGraph, bf_search, df_search
```

Comment:
There is a small chance that people used those two functions via ``from Bio.Pathway import bf_search`` or similar, but a workaround is easy.

Biopython has not typically defined ``__all__`` which would make this kind of thing explicit.